src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -13,4 +13,35 @@
     <Compile Include="System.Runtime.CompilerServices.Unsafe.il" />
     <Reference Include="System.Runtime" />
   </ItemGroup>
+
+  <Target Name="AddResourcesFileToIlasm" BeforeTargets="CoreCompile" Condition="'$(OS)'=='Windows_NT'">
```

Comment:
Were you able to test what ILDasm did on linux?  I'm curious if it does any better or this scenario (resource roundtripping) is completely broken.

Reply:
Actually I tried running it on Linux and ildasm runs succesfully, except that it only proudces the `.il` file, it won't produce the .res as I believe it is expected.